pom.xml

```diff
@@ -696,6 +705,26 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>io.confluent</groupId>
+                <artifactId>resolver-maven-plugin</artifactId>
+                <version>1.0.0-SNAPSHOT</version>
```

Comment:
Is there a plan for pinning to a concrete version? It certainly makes sense to go through a round of testing with SNAPSHOTs first, but will there be a follow-up that pins this to a specific version?

Reply:
I do want to release it, but since it will be public, I also need to publish to maven central as well, not sure if we have a process to do it?